pyproject.toml

```
@@ -43,7 +43,7 @@ dependencies = [
     "networkx>=3.3, < 4.0",
     "numpy >= 1.25.2, < 2.0",
     "qiskit >= 1.0, < 2.0",
-    "qiskit-iqm >= 15.1, < 16.0",
+    "qiskit-iqm > 17.0",
```

```suggestion
    "qiskit-iqm > 17.0, < 18.0",
```

Highly recommend to limit the upper version. Otherwise at some point the setup will break due to non-backwards-compatible update of Qiskit-on-IQM.

Also, did you mean specifically `> 17.0`, meaning `17.0` is not acceptable, it must be **higher** than 17.0?

---

Thank you for the comment, you're right about the correct limits. It should be "qiskit-iqm >= 17.0, < 18.0",